lib/benchmark_driver/repeater.rb

```diff
@@ -59,6 +61,39 @@ def average_result(values, rest_on_average)
         end
       end
 
+      def median_result_i(a)
+        b = a.dup
+        b.sort!
+        x = y = nil
+        until b.empty? do
+          x = b.pop
+          y = b.shift
+        end
+        x || y
```

Comment:
I wonder, why not the simpler `a.sort[a.size/2]` ?

Reply:
I agree that it should also work, thanks https://github.com/benchmark-driver/benchmark-driver/commit/4c927d259b8c7a41736a959fba8d91a12f12f9d5